### DISCUSSION TITLE
New integration: Whimsical

### DISCUSSION THREAD

**Discussion by jbergstroem** [outline/outline#2855] [Ideas]
[Whimiscal][1] is a tool that creates pretty mind maps, flowcharts, diagrams and so on. They tend to look good due to default styling & sizing and quickly grew as a team favorite in my company. They already have an embed function:
<img width="461" alt="Screen Shot 2021-12-13 at 3 12 36 PM" src="https://user-images.githubusercontent.com/176984/145865961-ab7fbba7-9216-46c4-a8ba-cafde06456b4.png">

..which should fit pretty well into an outline format (I use it frequently with hackmd.io).

[1]: https://whimsical.com

**Comment by tommoor** [comment]
Hi Johan, support for Whimsical has now been added – thanks for the request. As you mentioned it looks great within Outline as the design aesthetic is very similar.

**Reply by jbergstroem** [reply]
Impressive turnaround! Thanks a lot.

**Comment by aaptalking** [comment]
So, it now supported in Outline ? any update ?

**Reply by Limezy** [reply]
Yes https://www.getoutline.com/integrations/whimsical
